### ISSUE TITLE
TouchUI does not show new HTML in EEPROM Marlin Plugin v1.2+

### ISSUE THREAD

**Issue by amoose136** [BillyBlaze/OctoPrint-TouchUI#214]
As per the changes here: https://github.com/amsbr/OctoPrint-EEPROM-Marlin/compare/1.0.4...master the Marlin EEPROM editor plugin v1.2 is now in a tabbed based layout that is incompatible TouchUI. I'd be happy to fix this myself and submit a pull request except I don't really know how TouchUI works and I don't have a lot of guesses as to why TouchUI won't display the tabs so it will take me a quite a while to figure it out.

**Comment by amoose136** [comment]
A little digging reveals that it's caused by the same thing that hides the settings list. 
From touchui.css:
```css
#touch body .modal-scrollable .modal .modal-body .span3.scrollable, #touch body .modal-scrollable .modal .modal-body .nav.nav-pills {
    display: none;
}
```
In this case the `.nav.nav-pills` selector is selecting the ul that contains the tabs. Adding 
```css
#settings_plugin_eeprom_marlin+div>ul.nav.nav-pills {
  display: block;
}
```
Somewhere afterward will probably fix it but I can't confirm because Chrome's on-the-fly css editing tools are borked up/don't work right and I don't want to set up Firefox right now.

**Comment by amoose136** [comment]
```css
#settings_plugin_eeprom_marlin>div>ul.nav.nav-pills{
    display: block;
}
```
Is what is actually needed.

**Comment by BillyBlaze** [comment]
Thanks for reporting and finding out the issue. I will add your code from above comment into the upcoming release!

**Comment by BillyBlaze** [comment]
Thanks, this has been released in 0.3.12
